src/JavaCodeGeneration.jl

```diff
@@ -15,6 +15,46 @@ function generateconvertarg(x::Tuple{Int64, ClassDescriptor})
 end
 
 function methodfromdescriptors(
+    classdescriptor::ClassDescriptor,
+    methoddescriptor::MethodDescriptor
+)   
+    methodfromdescriptors(
+        Val(isstatic(methoddescriptor)),
+        classdescriptor,
+        methoddescriptor
+    )
+end
+
+function methodfromdescriptors(
+    ::Val{true},
+    classdescriptor::ClassDescriptor,
+    methoddescriptor::MethodDescriptor
+)   
+    paramtypes = map(paramexprfromtuple, enumerate(methoddescriptor.paramtypes))
+    signature = string(
+        methoddescriptor.rettype.signature,
+        '(',
+        map(x->x.signature, methoddescriptor.paramtypes)...,
+        ')')
+    body = quote
+        args = jvalue[]
+        $(map(generateconvertarg, enumerate(methoddescriptor.paramtypes))...)
+        result = callinstancemethod(
+            $(classdescriptor.jnitype),
+            $(QuoteNode(Symbol(methoddescriptor.name))),
```

Comment:
`QuoteNode` won't evaluate `methoddescriptor.name`

Reply:
I think we need it, because we want to quote the generated symbol. The code evaluates to:
```Julia
callinstancemethod(obj, :notifyAll, jvoid, "()V", args...)
```
otherwise we would have
```Julia
callinstancemethod(obj, notifyAll, jvoid, "()V", args...)
```
and it would be interpreted as a variable.
However, I will test this when we can finally perform the lazy generation.

Reply:
My mistake, I thought QuoteNode would result in `:methoddescriptor.name`